lib/lib.js

```diff
@@ -0,0 +1,201 @@
+var util = require('util');
+var os = require('os');
+
+var Transform = require('stream').Transform;
+
+var lstream = require('lstream');
+var request = require('request');
+var async = require('async');
+
+var github = new (require('github'))({
+  version: '3.0.0',
+  headers: {
+    'user-agent': 'git-apply-pr',
+  },
+});
+
+
+function authenticate(token) {
+  github.authenticate({
+    type: 'token',
+    token: token,
+  });
+}
+
+exports.getOutput = getOutput;
+function getOutput(type, config) {
+  var OUTPUT = {
+    'PR': util.format('#%d', config.PR),
+    'PR-URL': util.format('https://%s/%s/%s/pull/%d',
+      config.HOST, config.OWNER, config.REPO, config.PR),
+    'Reviewed-By': [],
+  };
+
+  return OUTPUT[type];
+}
+
+exports.getComments = getComments;
+function getComments(config, page, next) {
+
+  async.parallel({
+    issues: function getIssueComments(cb) {
+      github.issues.getComments({
+        user: config.OWNER,
+        repo: config.REPO,
+        number: config.PR,
+        per_page: 100,
+        page: page,
+      }, cb);
+    },
+    pr: function getPrComments(cb) {
+      github.pullRequests.getComments({
+        user: config.OWNER,
+        repo: config.REPO,
+        number: config.PR,
+        per_page: 100,
+        page: page,
+      }, cb);
+    }
+  },
+  function gotComments(err, comments) {
+    if (err) {
+      exitWithMsg(err);
+    }
+
+    var reviews = {};
+    comments = comments.issues.concat(comments.pr);
+    comments.forEach(function (comment) {
+      if (/lgtm/i.test(comment.body) ||
+          maybeTestForPlusone(config.ENABLE_PLUS_ONE, comment.body)) {
+        var val = reviews[comment.user.login] || 0;
+        reviews[comment.user.login] = val + 1;
+      }
+    });getOutput
+
+    if (comments.length < 100)
+      next(reviews, config);
+    else
+      getComments(page + 1, next);
+  });
+}
+
+function maybeTestForPlusone(ENABLE_PLUS_ONE, comment) {
```

Comment:
I would think that instead of taking `ENABLE_PLUS_ONE`, `maybeTestForPlusone` would take a configuration object and test for `config.ENABLE_PLUS_ONE`. I would also rename `maybeTestForPlusone` to `maybeTestForPlusOne` (capitalizing `One`).